pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/operations/security/index.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: layout.pug
+navigationTitle: Security
+excerpt: Security for DC/OS Apache HDFS service
+title: Security
+menuWeight: 50
+model: /mesosphere/dcos/services/hdfs/data.yml
+render: mustache
+---
+
+# DC/OS {{ model.techName }} Security
+
+The DC/OS {{ model.techName }} service supports {{ model.techShortName }}'s native transport encryption, authentication, and authorization mechanisms. The service provides automation and orchestration to simplify the usage of these important features.
+
+A good overview of these features can be found [here](https://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-common/SecureMode.html).
+
+<p class="message--note"><strong>NOTE: </strong>These security features are only available on DC/OS Enterprise 1.10 and later.</p>
+
+## Transport Encryption
+
+#include /mesosphere/dcos/services/include/security-transport-encryption-lead-in.tmpl
+
+<p class="message--note"><strong>NOTE: </strong>Enabling transport encryption is not <strong>required</strong> to use <a href="#kerberos-authentication">Kerberos authentication</a>, but transport encryption <strong>can</strong> be combined with Kerberos authentication.</p>
+
+#include /mesosphere/dcos/services/include/security-configure-transport-encryption.tmpl
+
+#include /mesosphere/dcos/services/include/security-transport-encryption-clients.tmpl
+
+## Authentication
```

Comment:
As mentioned below, please add a section for DNS forwarding and custom domain here. Then link this part of the doc back to the release notes.

Reply:
Done